bootstrap-pgmon.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+sudo yum -y groupinstall 'Development Tools'
+sudo yum -y install java-1.7.0*
+sudo yum -y install yum localinstall http://yum.postgresql.org/9.4/redhat/rhel-6-x86_64/pgdg-centos94-9.4-1.noarch.rpm
+sudo yum -y install postgresql9*
+sudo yum -y install tree
+sudo yum -y install vim
+
```

Comment:
@olivermel Please make sure only the packages actually required to build are listed here (not vim, tree, etc.)